web/management/commands/create_test_data.py

```
@@ -120,100 +132,116 @@
                 continue
 
             challenge = Challenge.objects.create(
-                title=f"Weekly Challenge {i + 1}",
-                description=f"Description for challenge {i + 1}",
+                title=f"Write a short poem",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)